doc/build-doc.sh

```diff
@@ -26,6 +26,9 @@ mkdir $SAMPLES_DIR
 # copy jupyter notebooks
 cd ..
 cp examples/notebooks/*.ipynb doc/$SAMPLES_DIR
+if [[ -f "doc/$SAMPLES_DIR/daal4py_data_science.ipynb" ]]; then
+    rm "doc/$SAMPLES_DIR/daal4py_data_science.ipynb"
+fi
```

Comment:
It would be better not to copy the file at all.
Something like that should work:
```suggestion
rsync -a --exclude='doc/$SAMPLES_DIR/daal4py_data_science.ipynb' examples/notebooks/*.ipynb doc/$SAMPLES_DIR
```